mmdeploy/codebase/mmaction/deploy/video_recognition.py

```diff
@@ -147,15 +147,17 @@ def init_pytorch_model(self,
 
     def create_input(self,
                      imgs: Union[str, np.ndarray, Sequence],
-                     input_shape: Sequence[int] = None) \
-            -> Tuple[Dict, torch.Tensor]:
+                     input_shape: Optional[Sequence[int]] = None,
+                     backend: Optional[Backend] = None,
```

Comment:
Why don't we move process_model_config into the task class to avoid add another parameter?

Reply:
Yes, it can be moved into task. But the parameter `backend` is to make difference between `creat_input`s of visualize pytorch model and backend model.